README.md

```diff
@@ -171,10 +169,20 @@ return <div>{example}</div>
 
 ### Add to your project
 
-- `yarn install next-translate`
+- `yarn add next-translate`
 
 **Note**: For a Next.js version below than `9.3.0`, use `next-translate@0.9.0` or below
 
+In your **package.json**:
```

Comment:
Hey @jvgranados! Thank for your contribution. However, this part of the README is explaining how to "getting started" with a custom server. In the case of the custom server, the idea is that instead of the CLI to transform your pages it's using the [i18nMiddleware](https://github.com/vinissimus/next-translate#i18nmiddleware). So, the `next-translate && next build` is an unnecessary step if you already have a middleware on the server.

Reply:
Hi @aralroca! My apologies, I thought the script would be necessary cause it appears in the [nextjs](https://nextjs.org/docs/advanced-features/custom-server) official documentation

Reply:
Don't worry. Maybe we need to clarify it in the docs if it is not clear!! 😄

The scripts yes, but the `next-translate` CLI command before `node server.js` and `next build` no, because in the case of a custom server the languages routes are managed by the `i18nMiddleware`